jdi-light/src/test/java/io/github/epam/properties/ScreensFolderTests.java

```diff
@@ -0,0 +1,46 @@
+package io.github.epam.properties;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.security.InvalidParameterException;
+import java.util.Properties;
+
+import static com.epam.jdi.light.common.Property.SCREENS_FOLDER;
+import static com.epam.jdi.light.common.PropertyValidationUtils.validateProperties;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class ScreensFolderTests {
+
+    @DataProvider
+    public static Object[] negativeData() {
+        return new Object[]{"cd;\\fd", "c:\\f[d", "C:\\Program*Files\\tests", "C:\\Program Files\\tests"};
```

Comment:
Lets add empty string here "" in case user set the parameter but left value empty

Reply:
Done